proposals/vip_1.ts

```diff
@@ -0,0 +1,47 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const vip_1: ProposalDescription = {
+  title: 'VIP-1: Grant Param Admin Role',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0xf0b50f04623eeaacfa1f202e062a3001c925a35c6b75d6903e67b43f44bbf152', // PARAMETER_ADMIN
+        '0x2172861495e7b85edac73e3cd5fbb42dd675baadf627720e687bcfdaca025096' // ROLE_ADMIN
+      ],
+      description: 'Transfer PARAMETER_ADMIN role admin to ROLE_ADMIN'
```

Comment:
You also need to grant this Timelock the relevant roles? `ROLE_ADMIN`, GOVERNOR` etc

Reply:
this was a test vip, I should have indicated as much on the file name

Reply:
Does the timelock need the `ROLE_ADMIN` role?

Reply:
I can't find the `ROLE_ADMIN` anywhere in the Volt codebase.